convert2rhel/unit_tests/conftest.py

```diff
@@ -16,29 +16,6 @@
 else:
     from unittest import mock  # pylint: disable=no-name-in-module
 
-try:
-    from pytest_catchlog import CompatLogCaptureFixture
```

Comment:
Did we put it here for python 2.6 compatibility?

Reply:
Yeah, that is correct. Forgot to add a comment here in the PR mentioning it was for 2.6, and now it can be removed. 

I think I added that try-except during the F2F that happened in Michigan. 

But yeah, since we will not run this under 2.6 anymore, this code can be removed.